modify template to proper toml format

JinxCappa opened the pull request:
Existing config that is generated is not processed correctly with latest version of frp (v0.58.1) due to formatting.

**IMPORTANT: Please do not create a Pull Request without creating an issue first!**

_(Any change needs to be discussed before proceeding. Failure to do so may result in the rejection of the pull request)._

### Summary
This PR fixes the toml configuration generated in the configmap by the operator

### Type of Change
This PR fixes/implements the following **bugs/features**:

- #5 

<!-- What existing problem does the pull request solve? -->

### How has this been tested?
Proof: Can be provided if necessary

<!-- Demonstrate the code is solid. Example: The exact commands you ran and their output or/ screenshots. -->

### Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [x] My code follows the code style of this project.

yejiayu commented on pkg/client/utils/template.go:
```diff
@@ -43,9 +42,9 @@ transport.useEncryption = true
-localIP = {{ $upstream.TCP.Host }}
+name = "{{ $upstream.Name }}"
+type = "udp"
+localIP = "{{ $upstream.TCP.Host }}"
```
$upstream.UDP.Host

JinxCappa replied:
fixed other fields as well as they also referred to TCP

yejiayu commented on pkg/client/utils/template.go:
```diff
@@ -43,9 +42,9 @@ transport.useEncryption = true
+name = "{{ $upstream.Name }}"
+type = "udp"
+localIP = "{{ $upstream.TCP.Host }}"
 localPort = {{ $upstream.TCP.Port }}
```
$upstream.UDP.Port

yejiayu commented on pkg/client/utils/template.go:
```diff
@@ -43,9 +42,9 @@ transport.useEncryption = true
+type = "udp"
+localIP = "{{ $upstream.TCP.Host }}"
 localPort = {{ $upstream.TCP.Port }}
 remotePort = {{ $upstream.TCP.ServerPort }}
```
$upstream.UDP.ServerPort

Commit:
modify template to proper toml format

Existing config that is generated is not processed correctly with latest version of frp (v0.58.1) due to formatting.

Signed-off-by: Eddie Garcia <jinyx007@gmail.com>
